src/qumin/config/qumin.yaml

```diff
@@ -73,7 +73,12 @@ entropy:
                       # with any file, use to compute entropy heatmap
                       # with n-1 predictors, allows for acceleration on nPreds entropy computation.
   merged: False       # Whether identical columns are merged in the input.
-  stacked: False      # whether to stack results in long form
+  stacked: False      # Whether to stack results in long form
+  legacy: False       # Whether to use the legacy calculation. Slightly slower, but less options.
```

Comment:
Is legacy the same as setting all tokens (should be token_freq) options to False ?

Reply:
Why do we need to keep the legacy option ? If we have a new, better implementation, let's keep only that, perhaps ?